docs/source/developer_guide/tutorial.rst

```diff
@@ -187,20 +187,34 @@ Alternatively, using ``tox``::
 Adding a new Observable
 =======================
 
-Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should conform to the ``__call__`` signature of ``janus_core.helpers.janus_types.Observable``. For a user this can be accomplished by writing a function, or class also implementing a commensurate ``__call__``.
+Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should extend ``janus_core.processing.observables.Observable``. The abstract method ``__call__`` should be implemented to obtain the values of the observed quantity from an ``Atoms`` object:
 
-Built-in observables are collected within the ``janus_core.processing.observables`` module. For example the ``janus_core.processing.observables.Stress`` observable allows a user to quickly setup a given correlation of stress tensor components (with and without the ideal gas contribution). An observable for the ``xy`` component is obtained without the ideal gas contribution as:
+.. code-block:: python
+
+    @abstractmethod
+    def __call__(self, atoms: Atoms) -> list[float]
+
+These values are returned as a ``list[float]`` representing the dimensions of the observed value which may be returned for a slice of atoms individually. ``value_count`` should be overloaded to return the expected length returned by ``__call__``. This is so enough correlators may be spawned to track the correlated the values. Note the final correlation will be the average of the correlations across these dimensions and atom counts.
+
+For example the ``janus_core.processing.observables.Stress`` built-in returns the required stress components as calculated by ``Atoms.get_stress`` called upon the tracked atoms. Therefore ``__call__`` returns one ``float`` per component correlated. For example an observable representing the ``xy`` and ``zy`` components of stress computed across all odd-index atoms in some ``Atoms`` object can be constructed as follows:
 
 .. code-block:: python
 
-    Stress("xy", False)
+    s = Stress(components=["xy", "zy"], atoms_slice=(0, None, 2))
 
-A new built-in observables can be implemented by a class with the method:
+In this case ``s(atoms)`` will return 2 values.
```

Comment:
This could be expanded, maybe with an actual example of the use with `atoms`. E.g. naively:

```
In [31]: s = Stress(components=["xy", "zy"], atoms_slice=(0, None, 2))

In [32]: a
Out[32]: Atoms(symbols='NaClNaClNaClNaCl', pbc=True, cell=[5.64, 5.64, 5.64], spacegroup_kinds=..., calculator=MACECalculator(...))

In [33]: s(a)
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
Cell In[33], line 1
----> 1 s(a)

File ~/PSDI/janus-core/janus_core/processing/observables.py:208, in Stress.__call__(self, atoms)
    194 def __call__(self, atoms: Atoms) -> list[float]:
    195     """
    196     Get the stress components.
    197
   (...)
    206         The stress components in GPa units.
    207     """
--> 208     sliced_atoms = atoms[self.atoms_slice]
    209     sliced_atoms.calc = atoms.calc
    210     stresses = (
    211         sliced_atoms.get_stress(
    212             include_ideal_gas=self.include_ideal_gas, voigt=True
    213         )
    214         / units.GPa
    215     )

File ~/PSDI/venv_janus_11_new/lib/python3.11/site-packages/ase/atoms.py:1151, in Atoms.__getitem__(self, i)
   1149 atoms.arrays = {}
   1150 for name, a in self.arrays.items():
-> 1151     atoms.arrays[name] = a[i].copy()
   1153 atoms.constraints = conadd
   1154 return atoms

IndexError: arrays used as indices must be of integer (or boolean) type
```

(I get a similar, but different error for a trajectory I read in, rather than just one `Atoms` structure)

Reply:
this appears to be a bug I must have introduced when apply review comments

Reply:
Fixed, and also the vaf test case now tests using ```slice```, ```range```, ```list[int]```, ```StartStopStep``` as atom slices